### ISSUE TITLE
Control Panel menu link

### ISSUE THREAD

**Issue by elinw** [Joomla3-Admin-template/joomla-cms#63]
The admin menu link to the control panel doesn't have the drop downs.
Also the submenu that displays in control panel is missing Profile Edit. I wouldn't be against moving that link to a different menu though, it's kind of out of place among the super admin links even though in terms of code that is where it should go. It has to be somewhere everyone with login rights to the administrator has access to.

**Comment by kyleledbetter** [comment]
yup, i moved all those links out under the dropdowns under Settings and username in the top right. The previous menu structure didn't make since having Control Panel under Site
